Source/WebCore/rendering/RenderLayerBacking.h

```diff
@@ -245,7 +245,8 @@ class RenderLayerBacking final : public GraphicsLayerClient {
     bool shouldAggressivelyRetainTiles(const GraphicsLayer*) const override;
     bool shouldTemporarilyRetainTileCohorts(const GraphicsLayer*) const override;
     bool useGiantTiles() const override;
-    bool useCSS3DTransformInteroperability() const override;
+    // FIXME: Should this have a different name if we need to preserve this difference indefinitely?
+    bool useCSS3DTransformInteroperability() const override { return true; }
```

Comment:
I don't think the difference is intentional, and I *believe* all the callers of this function will happen on clients that are `RenderLayerBacking`.

I do see the value in separating the 'remove the preference' changes from anything that could be a functional difference though, but it'd be worth trying to remove this either in this patch or as a follow-up.

Reply:
Filed https://bugs.webkit.org/show_bug.cgi?id=271141 and will do.